### PR TITLE
Add Voxelab Proxima 6 non-Chitu version (svgx file format)

### PR DESCRIPTION
**Pull request by Qbort**
## What does the pull request do?

Adds support for Voxelab Proxima 6 resin printer, newer version without the Chitu motherboard that only supports svgx files. Tested on my printer and it works. 

## What is the current behavior?

N/A

## What is the updated/expected behavior with this PR?

Ability to generate files in prusa slicer and subsequent conversion to svgx by UVtools

## How was the solution implemented (if it's not obvious)?

Added new prusa slicer profile and new printer type.

### CODE REVIEW

**Comment by sn4k3** [comment]
The SVGX lacks of AA, please disable it on slicer and update file. `gamma_correction` must be 0

**Comment by sn4k3** [comment]
Also provide a link for the printer, I can't find it. And also a slicer capable of output this format with the printer preset avaliable.

**Comment by Qbort** [comment]
@sn4k3 apologies for the very long delay in reply. Thank you for the helpful comments - I am very new to 3d printing and their file formats so I definitely learned with those. 

I made the corrections to the .ini file as per your comments. I must admit I do not know how to check the original resolution of the thumbnail sizes. I originally used the same thumbnails as existing `Voxelab Proxima 6.ini` but modifying  as per your suggeston seems to still work in my printer (and looks better on the small lcd screen).

> Also provide a link for the printer, I can't find it. 

The Voxelab Proxima 6.0 is no longer for sale - the company removed it, but still provides support. A few reddit discussions [1-3] show that the company at some point decided to change the original motherboard (manufactured by Chitu) for a different (no-brand) one and this resulted in only svgx file format being supported as well as a different user interface in the screen. They kept the same model name even though this change was quite significant. For the PrusaSlicer config thought calling it "Voxelab Proxima 6 - SVGX" should be sufficiently clear. Happy to rename if you disagree.

> And also a slicer capable of output this format with the printer preset avaliable.

The slicer that comes with the printer (VoxelPrint) can be downloaded from Voxelab website [4]. Make sure to select "VoxelPrint" from the top filter bar as shown in image below. Note that UVtools is not happy with the files generated from that software (same error message as shown in issue #462 ).

image

[1] https://www.reddit.com/r/Voxelab_Proxima/comments/sd6l0g/chitubox_vs_voxelprint/
[2] https://www.reddit.com/r/Voxelab_Proxima/comments/tdn733/has_anyone_swapped_a_motherboard/
[3] https://www.reddit.com/r/Voxelab_Proxima/comments/v25i6f/if_you_want_fdg_file_compatibility_avoid_serial/
[4] https://www.voxelab3dp.com/download?category=18#downloadList

File: PrusaSlicer/printer/Voxelab Proxima 6 - SVGX.ini [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+display_mirror_x = 1
+display_mirror_y = 0
+display_orientation = landscape
+display_pixels_x = 2560
+display_pixels_y = 1620
+display_width = 130.56
+elefant_foot_compensation = 0.2
+elefant_foot_min_width = 0.2
+fast_tilt_time = 5
+gamma_correction = 1
```
[the remark this answers is not in the document]
Comment (Qbort) [reply PrusaSlicer/printer/Voxelab Proxima 6 - SVGX.ini]: done

File: PrusaSlicer/printer/Voxelab Proxima 6 - SVGX.ini [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+printer_notes = Don't remove the following keywords! These keywords are used in the "compatible printer" condition of the print and filament profiles to link the particular print and filament profiles to this printer profile.\nPRINTER_VENDOR_PRUSA3D\nPRINTER_MODEL_SL1\nPRINTER_VENDOR_VOXELAB\nPRINTER_MODEL_PROXIMA_6_SVGX\nFILEFORMAT_SVGX\n\nSTART_CUSTOM_VALUES\nLightOffDelay_0\nBottomLightOffDelay_0\nBottomLiftHeight_5\nLiftHeight_5\nBottomLiftSpeed_60\nLiftSpeed_60\nRetractSpeed_150\nBottomLightPWM_255\nLightPWM_255\nEND_CUSTOM_VALUES
+printer_settings_id = 
+printer_technology = SLA
+printer_variant = default
+printer_vendor = 
+printhost_apikey = 
+printhost_cafile = 
+relative_correction = 1,1
+slow_tilt_time = 8
+thumbnails = 400x300,200x125
```
[the remark this answers is not in the document]
Comment (Qbort) [reply PrusaSlicer/printer/Voxelab Proxima 6 - SVGX.ini]: done (note I am not sure how to check which is original thumbnail size but confirmed it works in my printer)

[the remark this answers is not in the document]
Comment (sn4k3) [reply PrusaSlicer/printer/Voxelab Proxima 6 - SVGX.ini]: After install the slicer I have confirmed the thumbnails, the first is 54x54 and second is 200x240. However UVtools will force 128x128,200x240 due the forced sizes. How does thumbnail shows on your printer? Good, distorced, medium?

**Comment by sn4k3** [comment]
This product is a really a mess, they should not have the same name, but we know how these guys work, can't say it's a surprise as I have seen worse.

Well, I found on a website that the display is 50um pixel size and if so your display width and height is slight incorrect. To respect the 50x50um it should be:

```
Width: 0,05*2560 = 128mm
Height: 0,05*1620 = 81mm
```

You current have set to: 51um
To confirm this settings you should slice a cube like 40x40x5, print and measure with a caliper and confirm if the size match. Ideally you print two cubes and compare them (One with 51um, other with 50um)

**Comment by Qbort** [comment]
Happy New Year and thanks for the detailed analysis! I will try printing the 40x40x5mm block in the next few days. I had assumed the same resolution with the old Voxelab Proxima 6 - from my understanding the UV screen is the same - they just changed the motherboard (see Voxelab Proxima 6.ini). It is worth confirming nonetheless.

**Comment by sn4k3** [comment]
I'm merging this. We can fix it latter if the case.

**Comment by Qbort** [comment]
Thanks for merging. Sorry for the lack of response.

I've been struggling to figure out if there is anything I can do to print the test cube reliably. I noticed after printing some mechanical parts that the inner dimensions (holes) of the print were smaller than specified in my CAD file and the outer dimensions were bigger. 

There is an issue with my printer (I got it for free): there is a thin layer of resin over the LCD screen (spilt by the previous owner). I guess this might result in light bleed, so I can't tell reliably what the pixel size is supposed to be...

**Comment by sn4k3** [comment]
No problem... In any case you can confirm it latter please tell me.
Regards

**Pull request merged**